test/sanity/code-smell/docs-build.json

```diff
@@ -1,5 +1,6 @@
 {
     "disabled": true,
     "no_targets": true,
-    "output": "path-line-column-message"
+    "output": "path-line-column-message",
+    "maximum_python_version": "3.11"
```

Comment:
This change is no longer needed, now that the `docs-build` sanity test has been removed.